src/Catalyst.Core.Modules.Mempool.Tests/UnitTests/MempoolTests.cs

```diff
@@ -24,62 +24,63 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using Catalyst.Abstractions.Mempool.Repositories;
+using Catalyst.Abstractions.Hashing;
+using Catalyst.Abstractions.Mempool.Services;
 using Catalyst.Core.Lib.DAO;
+using Catalyst.Core.Lib.DAO.Transaction;
 using Catalyst.Core.Lib.Extensions;
-using Catalyst.Core.Lib.Extensions.Protocol.Wire;
+using Catalyst.Core.Modules.Hashing;
+using Catalyst.Protocol.Transaction;
 using Catalyst.Protocol.Wire;
 using Catalyst.TestUtils;
 using FluentAssertions;
 using Nethermind.Dirichlet.Numerics;
 using NSubstitute;
 using NSubstitute.ExceptionExtensions;
-using TheDotNetLeague.MultiFormats.MultiBase;
+using TheDotNetLeague.MultiFormats.MultiHash;
 using Xunit;
 
 namespace Catalyst.Core.Modules.Mempool.Tests.UnitTests
 {
     public sealed class MempoolTests
     {
+        private readonly IHashProvider _hashProvider;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_hashProvider' or refactor the code to use its value.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4941742)